### DISCUSSION TITLE
dial tcp 127.0.0.1:54588: connect: connection refused

### DISCUSSION THREAD

**Discussion by 2054686334** [nonebot/discussions#152] [问答]
运行持续报错，nonebot版本2.0.0b2，使用nonebot_plugin_gocqhttp，之前使用gocqhttp时也会报错，端口正确地址正确，不知道到底是什么原因导致的...

**Comment by mnixry** [comment]
应该是没注册Adapter

你可以试试更新一下nonebot_plugin_gocqhttp，可能更新完了以后启动都启动不起来
